cpp/tests/partitioning/hash_partition_test.cpp

```diff
@@ -414,8 +414,7 @@ TEST_F(HashPartition, StructofStructWithNulls)
     std::vector<std::unique_ptr<cudf::column>> s1_children;
     s1_children.emplace_back(s2.release());
     s1_children.emplace_back(c.release());
-    auto const null_it = nulls_at({3});
-    return structs_col(std::move(s1_children), std::vector<bool>{null_it, null_it + 6});
```

Comment:
Note: There are several API in the `test::iterators` namespace. `nulls_({....})` is used for specifying multiple nulls while `null_()` is for specifying just one null.

Reply:
Yes, my apologies for not going over those closer. I usually pick these things from other tests and use them, but I should definitely be picking the more correct version of these utilities.